modules/threescale-istio-adapter-apicast.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/threescale_adapter/threescale-adapter.adoc
```

Comment:
This looks like a new module.
1 - Do you want to change the file name to ossm-threescale-itstio-adapter-apicast so that the file shows up in the list with your other files?
2 - Since this is a new file, it is not included in service_mesh/v1x/threescale_adapter/threescale-adapter.adoc